pajbot/web/utils.py

```diff
@@ -70,7 +71,9 @@ def download_logo(client_id, streamer):
 
     twitchapi = TwitchAPI(client_id)
     try:
-        data = twitchapi.get(["users", streamer], base="http://127.0.0.1:7221/kraken/")
+        data = twitchapi.get(
+            ["users", streamer], base=f"http://{os.environ.get('KRAKEN_URL', '127.0.0.1:7221')}/kraken/"
```

Comment:
Also, a `KRAKEN_URL` should contain the `http://` and `/kraken` bits too, if you just want to ship kraken domain and port then it should be `KRAKEN_HOST` or `APIPROXY_HOST`